marklogic-data-hub-central/ui/src/components/sidebar/sidebar.tsx

```diff
@@ -495,7 +495,7 @@ const Sidebar: React.FC<Props> = (props) => {
       <Collapse
         className={styles.sideBarFacets}
         activeKey={activeKey}
-        expandIcon={panelProps => <Icon type="down" rotate={panelProps.isActive ? 0 : 180} data-testid='toggle' />}
+        expandIcon={panelProps => <Icon style={{ fontSize: '14px' }} type="down" rotate={panelProps.isActive ? 0 : -90} data-testid='toggle' />}
```

Comment:
Let's move the CSS to a class in the module.scss file and refer here using className.